src/core/lib/event_engine/posix_engine/lockfree_event.cc

```diff
@@ -141,8 +142,10 @@ void LockfreeEvent::NotifyOn(PosixEngineClosure* closure) {
         // contains a pointer to the shutdown-error). If the fd is shutdown,
         // schedule the closure with the shutdown error
         if ((curr & kShutdownBit) > 0) {
+          ABSL_ANNOTATE_IGNORE_READS_BEGIN();
```

Comment:
Why do we want to ignore races on the following line?

Reply:
Removed for now. But might need to add it back later

Reply:
If we do consider reintroducing this, I think we should have air-tight reasons for intentionally ignoring races. As in, strong confidence that no race could be problematic, or justification like "this is intentionally a racey read because <X>."